backend/people/migrations/0010_fill_staff_data_again.py

```diff
@@ -1,6 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
`git checkout master backend/people/migrations/0010_fill_staff_data_again.py` to revert all changes of this file.